web/src/components/Link.tsx

```diff
@@ -0,0 +1,12 @@
+export function Link({ href, linkText }: { href: string; linkText: string }) {
+  return (
+    <a
+      href={href}
+      target="_blank"
+      rel="noopener noreferrer"
+      className="text-sm font-semibold text-emerald-800 underline underline-offset-2 dark:text-emerald-500"
+    >
+      {linkText}
```

Comment:
I would prefer to use `children` here instead, so we're less restrictive on what can be entered and usage feels more natural IMO (e.g. `<Link href="#">My link text</Link>`)

Reply:
Good point!